document/deploy_docker_dev.md

```diff
@@ -1,6 +1,6 @@
 # Containerized Development Environment
```

Comment:
Consider add `Requirements` section in this instruction. (the user needs docker-compose)

Reply:
Done!